dist/tools/buildsystem_sanity_check/check.sh

```diff
@@ -109,9 +109,9 @@ UNEXPORTED_VARIABLES+=('JLINK_DEVICE' 'JLINK_IF')
 UNEXPORTED_VARIABLES+=('JLINK_PRE_FLASH' 'JLINK_RESET_FILE')
 UNEXPORTED_VARIABLES+=('GIT_CACHE' 'GIT_CACHE_DIR')
 UNEXPORTED_VARIABLES+=('LINKXX')
+UNEXPORTED_VARIABLES+=('APPDEPS' 'BUILDDEPS' 'DEBUGDEPS')
```

Comment:
'BUILDDEPS' 'DEBUGDEPS' seem unrelated?

Reply:
Yes but they are also not exported. I added them here as a guard, just in case. I can add them in a separate commit if you prefer?

Reply:
Yes, I would prefer a different commit